RELEASE.md

```diff
@@ -11,6 +11,7 @@ Please follow the established format:
 ## Bug fixes and other changes
 
 - Visual fixes for metrics plots in experiment tracking. (#1229)
+- Fix bug causing user to be unable to start Kedro-Viz if undefined parameters are used. (#1231)
```

Comment:
```suggestion
- Fix bug that prevented Kedro-Viz from loading when a project contains undefined parameters. (#1231)
```

Reply:
`contains` to `contained` to be past-tense consistent?

Reply:
Changed in [cf96f46](https://github.com/kedro-org/kedro-viz/pull/1231/commits/cf96f463092a0279b45e1e56d247576937416d1f).